config/routes.rb

```diff
@@ -26,6 +26,7 @@
     delete '/', on: :collection, action: :destroy
     get 'merged', on: :collection
   end
+  resources :bookable_slots, only: :index
   get '/bookable_slots/available', to: 'bookable_slots#available'
```

Comment:
Not overly fussed but maybe this declaration could be nested within the `resources` one above now?